src/ucp/api/ucp.h

```diff
@@ -144,7 +144,8 @@ enum ucp_feature {
                                            operations support */
     UCP_FEATURE_WAKEUP = UCS_BIT(4),  /**< Request interrupt notification
                                            support */
-    UCP_FEATURE_STREAM = UCS_BIT(5)   /**< Request stream support */
+    UCP_FEATURE_STREAM = UCS_BIT(5),  /**< Request stream support */
+    UCP_FEATURE_AM     = UCS_BIT(6)   /**< Request am support */
```

Comment:
@yosefe How would you like to handle the feature bit? I do not know how it could go into the experimental header without causing potential confusion.